kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -20,13 +20,11 @@
     RepoAuthFlags,
     Version,
 )
-from kpops.component_handlers.kubernetes.model import KubernetesManifest
+from kpops.components.common.kubernetes_model import KubernetesManifest
```

Comment:
what's the reason for this move/rename?

Reply:
I wonder if the `KubernetesManifest` should be under `component_handlers`

Reply:
perhaps it some point when we would want to support CRDs it would make sense to make a separate module or namespace package such as `kpops.manifests`. it's ok like this for now

Reply:
moved everything under `kpops.manifests`